app/api/files/exportRoutes.ts

```diff
@@ -56,7 +56,7 @@ export default (app: Application) => {
 
         const results = await search.search(req.query, req.language, req.user);
         // eslint-disable-next-line camelcase
-        const { dateFormat, site_name } = await settings.get();
+        const { dateFormat = '', site_name } = await settings.get();
```

Comment:
userUtils.ts site_name its used on the very next line as a string, without defaulting to string the ts compiler warns that it can be undefined.

in this case the site_name is passed to another function that accepts a string and is defaulted there, but the variable its not used in the context of this function.

https://github.com/huridocs/uwazi/blob/900f11a1b317eeb1fded4a6a640ed1eb027ce197/app/api/files/exportRoutes.ts#L67-L72